msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,203 @@
+| 5 | Teams -> Tab | Teams sends the tab access token to the tab as part of the result object returned by the `getAuthToken()` call. |
+| 6 | Tab app | The tab app parses the token using JavaScript to extract required information, such as the user's email address. The token returned to the tab app is both an access token and an identity token. |
+
+> [!IMPORTANT]
+> The `getAuthToken()` is only valid for consenting to a limited set of user-level APIs that is email, profile, offline_access, and OpenId. It is not used for further Graph scopes such as `User.Read` or `Mail.Read`. For suggested workarounds, see [Extend your app with Microsoft Graph permissions](tab-sso-graph-api.md).
+
+> [!NOTE]
+> Currently, SSO only supports OAuth 2.0 token and does not support SAML token.
+
+### Use cases for enabling SSO for tab
+
+Here are some use cases where enabling SSO is beneficial. Call `getAuthToken` in these scenarios to use Teams identity for obtaining access token for your app users:
+
+1. To get user’s teams identity from Teams if you have an existing app that you want to be available within a tab app in Teams for your app users.
+
+2. To authenticate a user by reusing the team’s identity inside your Tab application.
+
+3. To authenticate and get user’s teams identity without having them to signing in again inside configurable tabs. This is applicable to some settings need to be configured at an config stage.
+
+4. To obtain an access token inside a task module, when it's invoked from a tab, bots, messaging extension, or adaptive cards.
+
+5. To authenticate a user in stage view.
+
+The SSO API also works in [task modules](../../../task-modules-and-cards/what-are-task-modules.md) that embed web content.
+
+> [!NOTE]
+> Tabs are Teams-aware web pages. To enable SSO in a web page hosted inside a tab, add [Teams Javascript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), and call `microsoftTeams.initialize()`. Once you've initialized Microsoft Teams, call `microsoftTeams.getAuthToken()`, which will get the access token for your app.
+
+To achieve Teams SSO, you must configure your app to enable SSO for authenticating and authorizing your users.
+
+## Enable SSO for a tab app
+
+This section describes the tasks involved in implementing SSO for a tab app. These tasks are language- and framework-agnostic.
+
+To build a tab app that uses Teams SSO to authenticate users:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/teams-sso-accesstoken.png" alt-text="Steps to enable SSO for tab" border="false":::
+
+1. Register with Azure AD: Create an Azure AD app to generate an app ID and app ID URI You also configure redirect URI where Azure AD would send the access token in exchange for identity token for the current user logged into Teams.
```

Comment:
I think making "Register with Azure AD" bold would look much better similar to "Best practices" section.